src/common/scene-helpers.c

```diff
@@ -107,6 +107,14 @@ lab_wlr_scene_output_commit(struct wlr_scene_output *scene_output,
 		return false;
 	}
 
+	if (state->tearing_page_flip) {
+		if (!wlr_output_test_state(wlr_output, state)) {
+			wlr_log(WLR_DEBUG, "Output test for tearing failed on %s, "
+				"trying page-flip without tearing", wlr_output->name);
```

Comment:
I doubt we should fire a log message every time this fails, even in debug mode, especially when it could potentially fail with every single frame.

 If there is value in logging these failures at all, I would wrap the logging in a static function with some static counters every M seconds or N frames, whichever is most sensible and easiest to support.

Reply:
>  especially when it could potentially fail with every single frame.

Only when tearing is requested by a surface, which shouldn't happen all that often, mostly games (where the failures mostly happen in menus).

But I agree, I think we can just get rid of this log message.  The atomic backend will log as well when this check fails.

Reply:
Removed the log.